ipywidgets/interact.py

```diff
@@ -28,22 +37,40 @@ def _get_html(obj):
 
 class StaticInteract(object):
     """Static Interact Object"""
-    
+
     template = """
     <script type="text/javascript">
-      var mergeNodes = function(a, b) {{
+      let mergeNodes = function(a, b) {{
         return [].slice.call(a).concat([].slice.call(b));
       }}; // http://stackoverflow.com/questions/914783/javascript-nodelist/17262552#17262552
-      function interactUpdate(div){{
-         var outputs = div.getElementsByTagName("div");
-         //var controls = div.getElementsByTagName("input");
-         var controls = mergeNodes(div.getElementsByTagName("input"), div.getElementsByTagName("select"));
+
```

Comment:
I added some JS here. Essentially a label to the input element of type range which depicts the current value of the slider (similar to the one in ipywidgets).